code/modules/events/toilets.dm

```diff
@@ -1,19 +1,15 @@
 /datum/event/toilet_clog
```

Comment:
delete this event

Reply:
Because it's not needed

Reply:
foundation toilets apparently don't clog

Reply:
I remember removing clogging in general.

Reply:
Wow didn't catch that, toilet clogging was working back when I was able to play but apparently Tsuru disabled it. Removing event.

Reply:
If only it did anything, it would be fine. But it's just lag source and nothing more.

Reply:
I can't see an actual use for re-implementing clogging mechanics, especially since toilets have little-to-no in-game use, so it's no big deal.

Reply:
Although, if the plan was to remove clogging, why not just completely rip that code out?

Reply:
Because at that time I didn't feel like removing entire flood subsystem.